src/withTrackingComponentDecorator.js

```diff
@@ -117,7 +113,11 @@ export default function withTrackingComponentDecorator(
       }
 
       render() {
-        return <DecoratedComponent {...this.props} tracking={this.tracking} />;
+        return (
+          <ReactTrackingContext.Provider value={this.getContextForProvider()}>
```

Comment:
TODO: This will cause re-renders [per docs](https://reactjs.org/docs/context.html#caveats). Need to figure out a way around that...